src/api/index.md

```diff
@@ -2165,6 +2165,16 @@ type: api
   })
   ```
 
+  Sort using two keys:
+
+  ``` html
+  <ul>
+    <li v-for="user in users | orderBy ['lastName', 'firstName']">
```

Comment:
Sorry, didn't see the last comment.
That's fine, thanks for posting the commit :wink: